css/events/events.css

```diff
@@ -0,0 +1,136 @@
+.welcome-title {
+  margin-top: 50px;
+}
+
+.logo-img {
+  width: 150px;
+  display: block;
+  margin: 10px auto;
+}
+
+.social-media-logo {
+  width: 30px;
+  margin: 0px 4px;
+}
+
+.social-media-logo:hover {
+  transform: scale(1.1);
+}
+
+.event-details {
+  display: block;
+  text-align: center;
+}
+
+.event-title {
+  font-size: 4em;
+  font-weight: 400;
+  margin: 3px auto 3px;
+  color: firebrick;
+  font-family: sans-serif;
+}
+
+@media screen and (max-width: 640px) {
+  .event-title {
+    font-size: 2.7em;
+  }
+}
+@media screen and (max-width: 440px) {
+  .event-title {
+    font-size: 1.95em;
+    font-weight: 700;
+  }
+}
+
+@media screen and (min-width: 640px) {
+  .event-title {
+    font-size: 3.3em;
```

Comment:
Why are we following two different units for font-size, could you let me through any resources related to this decision